daemon/internal/newrelic/integration/test.go

```diff
@@ -763,25 +765,15 @@ func (t *Test) comparePhpPackages(harvest *newrelic.Harvest) {
 				len(expectedPackages), len(actualPackages), expectedPackages, actualPackages))
 			return
 		}
-		for i, _ := range expectedPackages {
```

Comment:
Why was the `_` removed?  Is it not needed to omit the value?

Also, why not just do 
`i, ePkg := range expectedPackages {`

then use `ePkg.Name` directly [here](https://github.com/newrelic/newrelic-php-agent/pull/1103/files#diff-c7f1af7665ab0865f15e79eeba7e73a7295fe66e0ff9205b9781c7b0d7c75faeR771)?

Reply:
Could you mark what is autoformatter?